src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -0,0 +1,28 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class LottoMachine {
+
+    private final LottoNumberGenerator generator;
+
+    public LottoMachine() {
+        this(new RandomLottoNumberGenerator());
+    }
+
+    public LottoMachine(LottoNumberGenerator generator) {
+        this.generator = generator;
+    }
+
+    public Lottos issue(Money purchaseAmount) {
+        int count = purchaseAmount.calculateLottoCount();
+
+        List<Lotto> lottos = IntStream.range(0, count)
+                .mapToObj(i -> new Lotto(generator.generate()))
+                .toList();
```

Comment:
저도 피드백을 받았던 부분이긴한데 IntStream.range(0,count)를 통해 단순히 반복 횟수를 제어하기 위해 Stream을 쓰는건 Stream이 가진 의미인 "데이터 흐름 중심으로 다룬다"보단 단순 반복문 대체 용도로 사용되어서 오히려 가독성이 떨어지는것 같습니다.
실제로 0,1,2... 값은 이 로직에서는 의미가 없고 단순 반복을 위해 사용되기 때문에
```suggestion
List<Lotto> lotts = new ArrayList<>();
  for(int i=0; i < count; i++){
    lottos.add(new Lotto(generator.generate()));
  }
return lotts;
```
이렇게 바꾸는게 "아 새로운 로또 count개 발행한다"라는 의도를 한번에 알 수 있을것 같습니다.
Stream이 틀린 방식이라는 의미가 아닙니다🥲(저도 정말 좋아합니다..)

Reply:
저도 성재님 코드 보면서 해당 리뷰를 봤습니다 🙂
말씀하신 대로 `Stream`은 데이터 흐름 중심의 의미로 사용될 때 더 자연스럽게 읽히는 것 같습니다.

각 메서드의 의도와 탄생 배경을 이해하고 사용하는 게 개발자 간 소통에도 훨씬 도움이 된다고 생각합니다 💡

Reply:
오 저는 무조건 stream이 좋다 생각했는데 아니군요.. 덕분에 배웁니다!